src/requirements/reqs-functions.ts

```diff
@@ -285,46 +214,135 @@ export function computeRequirements(
     reqs: computeUniversityRequirementFulfillments(coursesTaken)
   });
 
-  // PART 2: check college requirements
+  // PART 2: check college & major requirements
```

Comment:
It looks like this code also checks minor requirements, could you add that to this comment